index.html

```diff
@@ -9,57 +9,246 @@
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
-                    </li>
-                </ul>
-            </div>
+                <div class="letcon">
+                    <h3 class="let_connect">LET’S CONNECT</h3>
+                    <ul class="social-icons">
+                        <li>
+                            <a href="#"><img src="./assets/social/twitter.svg" alt="Twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/Linkedin.svg" alt="Linkedin"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="btn"><a href="#">Get resume</a></div>
+            </section>
+            <!-- ended -->
+
+            <section class="lang-container">
+                <div class="lang-title">
+                    <div class="h3-container">
+                        <h3>Languages</h3>
+                        <a href="#"><img src="./assets/Disabled.svg" alt=""></a>
+                    </div>
+                    <sdiv class="lang-group">
+                        <div class="lang js">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="lang html">
+                            <p>HTML</p>
+                        </div>
+                        <div class="lang css">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="about-frame">
+                    <div class="h3-container box">
+                        <h3>Framework</h3>
+                        <ion-icon class="ionicon" name="chevron-forward-outline"></ion-icon>
+                    </div>
+                    <div class="h3-container box">
+                        <h3>Skills</h3>
+                        <ion-icon class="ionicon" name="chevron-forward-outline"></ion-icon>
+                    </div>
+                </div>
+            </section>
         </section>
```

Comment:
- [ ] It is required that the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li. Kindly change that too.
- [ ] It is equally required that you must produce a close replica of the Figma design. Your design look differently to how it is on the Figma design. Please, take a look.

### Your Design
![your-lang](https://user-images.githubusercontent.com/26597432/236418028-3d176240-b43b-4d8f-b686-7ee4c7441a4f.png)

### Figma Design
![temp-lang](https://user-images.githubusercontent.com/26597432/236418156-8d05e250-f431-4a77-9323-31a2af09dce8.png)